packer/install-dependencies.sh

```diff
@@ -312,7 +312,10 @@ for KUBE_SVC_REDIRECT_VERSION in ${KUBE_SVC_REDIRECT_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
 done
 
-KV_FLEXVOLUME_VERSIONS="0.0.7"
+KV_FLEXVOLUME_VERSIONS="
+0.0.7
```

Comment:
Thanks for this PR @aramase! I don't think we need to keep the old version around. WDYT?

Reply:
Yeah, I don't think we need to keep it.